### PR TITLE
Create ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by DawidSAG

Ticket: [MTM-59579](https://cumulocity.atlassian.net/browse/MTM-59579)
Original PR: [7050](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7050)

[MTM-59579]: https://cumulocity.atlassian.net/browse/MTM-59579?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: fix: disable alarms refresh section when BE returns 403 status (#7050)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md]: The Claude AI suggested to use the following title:
```suggestion
title: Disable alarms refresh button when user lacks access to any alarms
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59579
+version: 1021.0.0
+---
+fix: disable alarms refresh section when BE returns 403 status (#7050)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md]: The Claude AI suggested to use the following body:
```suggestion
The alarms refresh section is now disabled when users lack permission to refresh alarms.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by DawidSAG — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59579
+version: 1021.0.0
+---
+The alarms refresh section is now disabled when users lack permission to refresh alarms.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md]: @DawidSAG What is meant by section? The ticket mentions auto-refresh and in the title it says "button". What is correct?

Comment (rahulc8y) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md]: Title says when user lacks "access to any alarms" but description says "permission to refresh alarms.". These should be aligned to clearly indicate the specific permission that is required by the user

Comment (DawidSAG) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md]: This task involves disabling an entire section of buttons when a user lacks access to alarms (Alarms list view).

image

Comment (DawidSAG) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md]: @BeateRixen @rahultsag what about:
title: Disable alarms refresh controls when user lacks access to alarms
and 
description: The auto-refresh controls and manual refresh button are disabled when user lacks access to alarms.

Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md]: Sounds good to me. But should be "The auto-refresh controls and the manual refresh button are now disabled when the user lacks access to alarms."

Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md]:
```suggestion
The auto-refresh controls and the manual refresh button are now disabled when the user lacks access to alarms.
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Disable alarms refresh button when user lacks access to any alarms
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-fix-disable-alarms-refresh-section-when-be-returns-403-status-7050.md]:
```suggestion
title: Disable alarms refresh controls when user lacks access to alarms
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**